src/crd2jsonschema.sh

```diff
@@ -65,7 +73,9 @@ function get_jsonschema_file_name()
     crd_kind="$(get_crd_kind "$crd")" || exit 1
     local crd_version
     crd_version="$(get_crd_version "$crd")" || exit 1
-    echo "${crd_kind}_${crd_version}.json"
+    local crd_group
+    crd_group="${get_crd_group "$crd")" || exit 1
```

Comment:
```suggestion
    crd_group="$(get_crd_group "$crd")" || exit 1
```
Hello @jacksgt. Thank you for the contribution. This typo should make it work.

Reply:
Oh indeed, well spotted! Fixed.